### ISSUE TITLE
readme.cmd really needs updating

### ISSUE THREAD

**Issue by keithl8041** [sayedihashimi/package-web#53]
It would be very handy just to link the readme.cmd file to the YouTube intro vid that you made;

http://sedodream.com/2012/03/14/PackageWebUpdatedAndVideoBelow.aspx

as well as;

http://sedodream.com/2011/12/24/PackageOncePublishAnywhere.aspx

**Comment by sayedihashimi** [comment]
I've updated readme.md with the links you specified. Let's leave this open as it still needs improving.

**Comment by okram999** [comment]
IMHO: A README file should be elaborate and easy to understand. And must require minimal searches to figure out one how to use the code/tool.

**Comment by sayedihashimi** [comment]
I agree but I've been swamped with other work so I haven't had tine to get to this.
